reactioncommerce/reaction
# Fix #4090: Scrolling to bottom adds more products to the view

Resolves #4090  
Impact: **major**  
Type: **bugfix**

## Issue
The current logic was checking for equality in the scroll position, the scroll value and the screen height. This usually works but is not true in some cases where the values are in decimal. There is a offset of `1`

## Solution
Instead of equality now the code checks for `+-1`.

## Testing
1. View store as anonymous.
2. Zoom out the browser.
3. Try to scroll, new products should get added.
